diff --git a/include/fruit/normalized_component.h b/include/fruit/normalized_component.h
index e9dca2f..366aef0 100644
--- a/include/fruit/normalized_component.h
+++ b/include/fruit/normalized_component.h
@@ -113,7 +113,7 @@ public:
   template <typename... FormalArgs, typename... Args>
   explicit NormalizedComponent(Component<Params...> (*)(FormalArgs...), Args&&... args);
 
-  NormalizedComponent(NormalizedComponent&& storage) noexcept : storage(std::move(storage.storage)) {}
+  NormalizedComponent(NormalizedComponent&&) noexcept = default;
   NormalizedComponent(const NormalizedComponent&) = delete;
 
   NormalizedComponent& operator=(NormalizedComponent&&) = delete;
